pom.xml

```diff
@@ -53,19 +59,19 @@
     </dependency>
     <dependency>
       <groupId>org.apache.spark</groupId>
-      <artifactId>spark-core_2.11</artifactId>
+      <artifactId>spark-core_2.12</artifactId>
       <version>${spark.version}</version>
     </dependency>
     <dependency>
       <groupId>org.json4s</groupId>
-      <artifactId>json4s-jackson_2.11</artifactId>
-      <version>3.2.11</version>
+      <artifactId>json4s-jackson_2.12</artifactId>
+      <version>3.5.3</version>
       <scope>compile</scope>
     </dependency>
     <dependency>
       <groupId>org.apache.crail</groupId>
       <artifactId>crail-client</artifactId>	
-      <version>1.2-incubating-SNAPSHOT</version>
+      <version>1.3-incubating-SNAPSHOT</version>
```

Comment:
Should we depend on a non-snapshot? Or are there any changes that require latest and greatest (since we do not push the SNAPSHOT to maven central this would require crail source to compile)